packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -141,7 +143,7 @@ export const components: ThemeOptions["components"] = {
   MuiBackdrop: {
     styleOverrides: {
       root: {
-        backgroundColor: "rgba(29,29,33,0.75)",
+        //backgroundColor: "rgba(29,29,33,0.75)",
```

Comment:
why are we commenting this ?

Reply:
I discovered a bug with `MuiBackdrop` not applying `MuiBackdrop-transparent` when using `Select`. Removing this property fixes that. I've made myself a ticket to find a real solution next sprint.

Because we don't have a token for this value, I wanted to keep it here until I can refactor next week.